### PR TITLE
ISACT ICBBank → ISBBank Indexing Error

### PR DESCRIPTION
**Pull request by reitowo**
https://github.com/ME3Tweaks/LegendaryExplorer/issues/394

### CODE REVIEW

**Comment by reitowo** [comment]
Sorry for overwhelming with formatting changes, probably you should save the Formatting Rule in the git to prevent this, as all contributor can format like you did.

Main changes:
https://github.com/ME3Tweaks/LegendaryExplorer/pull/395/files#diff-440ed65d7dce76819eb708b3dc02f0e823e11c20eea49ea16769b75b9f7d337dR1475-R1504

https://github.com/ME3Tweaks/LegendaryExplorer/pull/395/files#diff-b82b84982b83e64a5076fa0b2df6ad50042adeea5f4f61ab994b3acdf907fb6fL613-L662

File: LegendaryExplorer/LegendaryExplorer/Tools/AssetDatabase/AssetDatabaseWindow.xaml.cs [hunk 1]
```diff
@@ -2174,10 +2234,7 @@ private async Task dumpPackages(List<string> files, MEGame game)
             var elapsed = DateTime.Now - beginTime;
             MessageBox.Show(this, $"{CurrentGame} Database generated in {elapsed:mm\\:ss}");
             MemoryAnalyzer.ForceFullGC(true);
-            if (!CurrentGame.IsGame1())
-            {
-                GetConvoLinesBackground();
-            }
+            GetConvoLinesBackground();
```
Comment (Mgamerz) [inline LegendaryExplorer/LegendaryExplorer/Tools/AssetDatabase/AssetDatabaseWindow.xaml.cs]: Why is this changed?

Comment (reitowo) [reply LegendaryExplorer/LegendaryExplorer/Tools/AssetDatabase/AssetDatabaseWindow.xaml.cs]: I saw that function actually handles Game1 at start, and I just remove all Game1 restrictions like this. If this restriction was intended for something else, I'll add it back.

**Comment by Mgamerz** [comment]
Please remove the change to GetConvoLinesBackground(), as it does not have to do with this PR (this has to do with ISACT). I have committed this change separate.

**Comment by Mgamerz** [comment]
Hi, this has been merged. Thanks for the PR!
